### PR TITLE
created profile in the accounts context

### PR DESCRIPTION
**Pull request by CorneliaKelinske**
@alanvardy I have created the profile. Also set has_one in the user profile.
However, I was not able to set belongs_to in the profile file, since it is already associated through the user id.

### CODE REVIEW

File: lib/the_little_thinkers_space/accounts/profile.ex [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    field :user_id, :id
+
+    timestamps()
+  end
+
+  @doc false
+  def changeset(profile, attrs) do
+    profile
+    |> cast(attrs, [:full_name, :nickname, :birthday, :hobbies, :color, :animal, :food, :superhero, :song, :show, :book, :future, :joke])
+    |> validate_required([:full_name, :nickname, :birthday, :hobbies, :color, :animal, :food, :superhero, :song, :show, :book, :future, :joke])
```
Comment (alanvardy) [inline lib/the_little_thinkers_space/accounts/profile.ex:28]:
```suggestion
  @required_attrs [:full_name, :nickname, :birthday, :hobbies, :color, :animal, :food, :superhero, :song, :show, :book, :future, :joke]
  
  def changeset(profile, attrs) do
    profile
    |> cast(attrs, @required_attrs)
    |> validate_required(@required_attrs)
```

File: lib/the_little_thinkers_space/accounts/user.ex [hunk 2]
```diff
@@ -18,6 +18,8 @@ defmodule TheLittleThinkersSpace.Accounts.User do
     field :password, :string
     field :role, :string
 
+    has_one :profile, Accounts.Profile
```
Comment (alanvardy) [inline lib/the_little_thinkers_space/accounts/user.ex:21]: :+1:

File: lib/the_little_thinkers_space_web/controllers/profile_controller.ex [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+  def delete(conn, %{"id" => id}) do
+    profile = Accounts.get_profile!(id)
+    {:ok, _profile} = Accounts.delete_profile(profile)
+
+    conn
+    |> put_flash(:info, "Profile deleted successfully.")
+    |> redirect(to: Routes.profile_path(conn, :index))
+  end
+end
```
Comment (alanvardy) [inline lib/the_little_thinkers_space_web/controllers/profile_controller.ex:62]: Don't forget the auth! (I am sure you haven't)

**Review by alanvardy — APPROVED** [review]

File: lib/the_little_thinkers_space/accounts/profile.ex (hunk 1 above)
Comment (CorneliaKelinske) [reply lib/the_little_thinkers_space/accounts/profile.ex:28]: Thank you! Great idea!

**Pull request merged**